freqtrade/freqtradebot.py

```diff
@@ -480,6 +480,14 @@ def execute_buy(self, pair: str, stake_amount: float, price: Optional[float] = N
         else:
             # Calculate price
             buy_limit_requested = self.exchange.get_rate(pair, refresh=True, side="buy")
+            if self.config.get('use_custom_entry_price', False):
+                buy_rate = self.exchange.get_rate(pair, refresh=True, side="buy")
```

Comment:
Please reuse `buy_limit_requested` to avoid making another (unnecessary) call to the exchange.